src/ralph/api/auth/basic.py

```diff
@@ -145,15 +138,14 @@ def get_basic_auth_user(
     except StopIteration:
         # next() gets the first item in the enumerable; if there is none, it
         # raises a StopIteration error as it is out of bounds.
-        logger.warning(
+        logger.info(
             "User %s tried to authenticate but this account does not exists",
             credentials.username,
         )
         hashed_password = None
-    except AuthenticationError as exc:
-        raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN, detail=str(exc)
-        ) from exc
+    except AuthenticationError:
+        logger.info("Error while authenticating using Basic auth")
```

Comment:
If we want this log to be informative, we need more context to identify which user performed the action. WDYT?

Reply:
Reverted to the original exception, following @SergioSim 's comment